rdrf/rdrf/templates/registration/registration_base.html

```diff
@@ -114,10 +114,12 @@
 
         function onBeforeSubmit() {
             $("#id_email").val($("#id_username").val());
-            $("#patient-form").accordion("option", "active" ,0);
-            if ($("#guardian-form").length > 0) {
-                $("#guardian-form").accordion('option', 'active' ,0);
-            }
+            {% if not is_mobile %}
```

Comment:
I agree with your comment that the mobile special cases should be handled nicer than this.

What I would do here is create a function

```javascript

    function expandAccordion(formSelector) {
        if (! $(formSelector).length ) {
            return;
        }
        var accordion = $(formSelector).accordion("instance");
        if (typeof(accordion) === "undefined") {
            return;
        }
        accordion.option("active", 0);
    }
```

and then call it from `onBeforeSubmit()` like so:

```javascript
    expandAccordion("#patient-form");
    expandAccordion("#guardian-form");
```

`expandAccordion` would expand only if the form exists and accordion has been set up on it, which is what we really care about (not tying the decision to the special case of `is_mobile`)